### PR TITLE
Fix NonType error when importing google.api_core fails

### PR DESCRIPTION
**Pull request by ihji**
skipIf doesn't work for parameterized arguments since it's executed first.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by ihji** [comment]
R: @ahmedabu98
CC: @pabloem

File: sdks/python/apache_beam/io/gcp/bigquery_test.py [hunk 1]
```diff
@@ -858,19 +858,19 @@ class BigQueryStreamingInsertsErrorHandling(unittest.TestCase):
...
-          exception_type=exceptions.ServiceUnavailable,
+          exception_type=exceptions.ServiceUnavailable if exceptions else None,
           error_reason='backendError'),
       param(
-          exception_type=exceptions.InternalServerError,
+          exception_type=exceptions.InternalServerError if exceptions else None,
           error_reason='internalError'),
       param(
-          exception_type=exceptions.InternalServerError,
+          exception_type=exceptions.InternalServerError if exceptions else None,
```
Comment (ahmedabu98) [inline sdks/python/apache_beam/io/gcp/bigquery_test.py:873]: Do these tests still run if empty params are passed in? Would it be more optimal to skip them entirely? Something like this in the beginning of each affected test:
```
if exceptions is None:
  self.skipTest("Import fail: google.api_core")
```

Comment (ihji) [reply sdks/python/apache_beam/io/gcp/bigquery_test.py:873]: it's already there: https://github.com/apache/beam/blob/541dfb38105b2453665f9d16f2423456f043168d/sdks/python/apache_beam/io/gcp/bigquery_test.py#L851

this pr fixes the error thrown even when the test is marked as skipped.

Comment (ahmedabu98) [reply sdks/python/apache_beam/io/gcp/bigquery_test.py:873]: Oh I see. This LGTM then.

**Review by ahmedabu98 — APPROVED** [review]

**Pull request merged**